mftool.py

```diff
@@ -32,7 +32,7 @@ class Mftool:
     class which implements all the functionality for
     Mutual Funds in India
     """
-    def __init__(self):
+    def __init__(self, session=None, cache=None):

```

Comment:
Why do we require to pass a session and cache while creating an object if not None?
I guess it can be handled internally.